node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -487,9 +472,23 @@ impl Initialized {
 				?session,
 				"Importing dispute votes from chain for candidate"
 			);
+
+			let relay_parent = match overlay_db.load_candidate_votes(session, &candidate_hash)? {
```

Comment:
Uh sorry - I missed that. Having to fetch votes again just to retrieve the session info is kinda crap as well. Ideally we would refactor the code below to not even [need the `SessionInfo`](https://github.com/paritytech/polkadot/issues/3455#issuecomment-1157652087) ... which would be possible if we only stored the validator index. Anyhow that is certainly out of scope for this PR, let's just go back to passing in a relay parent :grimacing: